declarations.d.ts

```diff
@@ -1761,6 +1761,15 @@ interface INet {
 	 * @return {Promise<boolean>} true if the port is available.
 	 */
 	isPortAvailable(port: number): Promise<boolean>;
+
+	/**
+	 * Waits for port to be in LISTEN state.
+	 * @param {number} port Port to be checked.
+	 * @param {number} timeout Max amount of time in milliseconds to wait.
+	 * @param {number} inteval @optional The amount of time between each check.
+	 * @returns {boolean} true in case port is in LISTEN state, false otherwise.
+	 */
+	waitForPortToListen(port: number, timeout: number, interval?: number): Promise<boolean>;
```

Comment:
You can optionally introduce an interface here instead of accepting three different arguments